### DISCUSSION TITLE
关于  -g  --language  参数

### DISCUSSION THREAD

**Discussion by kdurov** [nxtrace/NTrace-core#77] [Ideas]
## 你正在使用哪个版本的 nexttrace？

 v1.0.8 linux amd64

## 你看到的异常现象是什么？

-g en 参数配合 -F 参数时无法切换英文

## 你期待看到的正常表现是怎样的？

` nexttrace -g en -F` 选择ISP菜单和追踪结果显示英语

## 请附上你的命令
` nexttrace -g en -F`

请问能否翻译完整 -F 选择ISP文本，还有追踪结果的geoip信息呢？
给不懂中文的朋友推荐这个时候发现的这个翻译不全。

谢谢。

**Comment by tsosunchia** [comment]
有一点疑问想问一下您，关于-F(即fast-trace模式)，它的功能是检查到中国大陆四网的路由，如果是不懂中文的外国朋友，真的会使用这个功能吗？或者说您能举个这方面应用的例子吗？

**Comment by kdurov** [comment]
会啊。
国内时候让国外朋友家里网搭个代理什么临时用下时候，会让对方看看到国内线路怎样。

**Comment by sjlleo** [comment]
您好，

Fast Trace 是给新手使用的功能，它本应该由 Shell 脚本来完成。

后面因为很多朋友的强烈要求下才集合在 NextTrace 中，但是我们认为它集成在 NT 中过于臃肿，因为它不属于 NT 应该有的功能，所以一度想要从中移除。目前 NT 的 Fast Trace 功能由网友自发维护，NT 项目组自身并不会涉及到该模块的开发。

这个和 traceroute、mtr、besttrace、worsttrace 中也没有集成 Fast Trace 是一样的道理，如果您对这个功能的多语言实现有兴趣，欢迎向我们提交相关 PR，我们会很乐意 Merge 这部分代码。

感谢

**Comment by kdurov** [comment]
不好意思。
不会代码，不过如果只是翻译文本可以帮看看。

**Comment by noctilust** [comment]
Suggest "-g" option defaults to "en", or add an option such as "-c" to read config file as such user could set default language in config file.

**Reply by tsosunchia** [reply]
我们在考虑加入一个环境变量来实现您的需求

**Reply by tsosunchia** [reply]
您目前可以先使用alias，
如：
alias nexttrace='nexttrace -g en'

**Comment by tsosunchia** [comment]
您需要的此功能，已经最近的更新中加入，待正式发版

**Reply by tsosunchia** [reply]
已发版
